tests/NuGet.Services.BasicSearchTests/StartupTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NuGet.Services.BasicSearchTests.Fixtures;
+using NuGet.Services.BasicSearchTests.Models;
+using Xunit;
+
+namespace NuGet.Services.BasicSearchTests
+{
+    public class StartupTests : IClassFixture<StartupTestsFixture>
+    {
+        private readonly StartupTestsFixture _fixture;
+
+        public StartupTests(StartupTestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public async Task AutocompleteParameterIsObserved()
+        {
+            // Arrange
+            var app = _fixture.App;
+            string query = "Id:angularjs Id:Antlr Id:WebGrease";
+
+            // Act
+            var withPrereleaseResponse = await app.Client.GetAsync("/query?" + Query(new { q = query, prerelease = true }));
+            var withPrerelease = await withPrereleaseResponse.Content.ReadAsAsync<QueryResult>();
+
+            var withoutPrereleaseResponse = await app.Client.GetAsync("/query?" + Query(new { q = query, prerelease = false }));
+            var withoutPrerelease = await withoutPrereleaseResponse.Content.ReadAsAsync<QueryResult>();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, withPrereleaseResponse.StatusCode);
+            Assert.Equal(HttpStatusCode.OK, withoutPrereleaseResponse.StatusCode);
+
+            Assert.Equal("1.2.0-RC1", withPrerelease.GetPackageVersion("angularjs"));
+            Assert.Equal("3.4.1.9004-pre", withPrerelease.GetPackageVersion("Antlr"));
```

Comment:
`Assert` can be extended but I don't think we should write our own xUnit. Perhaps add a line comment describing the assertion so that when you navigate to the failed assertion some extra info is available to the dev?